charts/jindofsx/templates/fuse/daemonset.yaml

```diff
@@ -64,7 +64,6 @@ spec:
       securityContext:
```

Comment:
Please also update chart version and changelog. thanks.

Reply:
Done